src/main/java/com/snowflake/kafka/connector/internal/InternalUtils.java

```diff
@@ -170,6 +186,17 @@ static Properties createProperties(Map<String, String> conf, boolean sslEnabled)
      */
     properties.put(SFSessionProperty.ALLOW_UNDERSCORES_IN_HOST.getPropertyKey(), "true");
 
+    if (ingestionMethodConfig == IngestionMethodConfig.SNOWPIPE_STREAMING) {
+      final String providedSFRoleInConfig = conf.get(Utils.SF_ROLE);
+      if (!Strings.isNullOrEmpty(providedSFRoleInConfig)) {
+        LOGGER.debug("Using provided role {} for JDBC connection.", providedSFRoleInConfig);
```

Comment:
Could we combine the log here with the one in the else branch?

Reply:
And I suggest to use INFO since DEBUG logging at this point is useless due to the large number of debug logging in the ingest SDK

Reply:
Yes, makes sense. no harm in using info.